docs/src/pages/components/icons/icons.md

```diff
@@ -7,61 +7,21 @@ components: Icon, SvgIcon
 
 <p class="description">Guidance and suggestions for using icons with Material-UI.</p>
 
-A [system icon](https://material.io/design/iconography/system-icons.html) or UI icon,
-symbolizes a command, file, device, or directory.
-System icons are also used to represent common actions like trash, print, and save,
-and are commonly found in app bars, toolbars, buttons, and lists.
-Google has provided a set of [Material icons](https://material.io/tools/icons/?style=baseline) that follow these guidelines.
+Material-UI provides icons support in three ways.
 
-Material-UI provides two components to render system icons: `SvgIcon` for rendering SVG paths, and `Icon` for rendering font icons.
+- Standardized Material Design icons are exported as React components from `@material-ui/icons` package (Material Icons).
+- With the `SvgIcon` component as a React wrapper for a custom SVG icon is exported from `@material-ui/core`.
+- With the `Icon` component as a React wrapper for custom font icons is exported from `@material-ui/core`.
 
-## SVG Icons
+## Material Icons (recommended)
 
-The `SvgIcon` component takes an SVG `path` element as its child and converts it to a React component that displays the path,
-and allows the icon to be styled and respond to mouse events. SVG elements should be scaled for a 24x24px viewport.
-
-The resulting icon can be used as is,
-or included as a child for other Material-UI components that use icons.
-By default, an Icon will inherit the current text color.
-Optionally, you can set the icon color using one of the theme color properties: `primary`, `secondary`, `action`, `error` & `disabled`.
-
-{{"demo": "pages/components/icons/SvgIcons.js"}}
-
-### SVG Material icons
+Material Design has standardized over 1,000 commonly needed icons, each in five different "themes" (see below). For each icon we export the respective React component from the `@material-ui/icons` package. Material Icons are based on SVG versions of Material Design icons. You can search the full list of these icons in [Material Icons](/components/material-icons).
```

Comment:
```suggestion
Material Design has standardized over 1,000 commonly needed icons, each in five different "themes" (see below). For each SVG icon we export the respective React component from the `@material-ui/icons` package. You can search the full list of these icons in [Material Icons](/components/material-icons).
```

Not sure if that suggestion was rejected or just lost?

Reply:
In this case I like my version a bit more. But if you think I should change I will do it :)

Reply:
It's a bit wordy, no disrespect.

Reply:
OK. I accept your suggestion then. :)